addons/armaos/CfgVehicles.hpp

```diff
@@ -20,6 +20,18 @@ class CfgVehicles
 			//init = "params ['_entity']; call compile preprocessFileLineNumbers '\z\ae3\addons\main\init\initLaptop.sqf';";
 		};
 
+		class AE3_Equipment
```

Comment:
Add interaction module to requiredAddons

Reply:
done